modules/aerodyn/src/AirfoilInfo.f90

```diff
@@ -477,9 +479,13 @@ SUBROUTINE ReadAFfile ( AFfile, NumCoefs, InCol_Alfa, InCol_Cl, InCol_Cd, InCol_
          ENDDO ! Row
 
       ENDIF
-
-
-         ! How many columns do we need to read in the input and how many total coefficients will be used?
+      
+      ! Reading Boundary layer file  for aeroacoustics
+      CALL ParseVar ( FileInfo, CurLine, 'BL_file' , p%BL_file , ErrStat2, ErrMsg2, UnEc )
+         CALL SetErrStat( ErrStat2, ErrMsg2, ErrStat, ErrMsg, RoutineName )
+      p%BL_file=trim(PriPath)//trim(p%BL_file)
```

Comment:
Does this work as intended when `BL_file` is specified using an absolute path?

Reply:
Good question.  I'll take a look at that.

Reply:
Confirmed your suspicion.  This does not work with absolute paths.  Fix in progress

Reply:
Fixed.